mne/io/eeglab/eeglab.py

```diff
@@ -656,13 +657,10 @@ def read_events_eeglab(eeg, event_id=None, event_id_func='strip_to_integer',
         eeg = io.loadmat(eeg, struct_as_record=False, squeeze_me=True,
                          uint16_codec=uint16_codec)['EEG']
 
-    if isinstance(eeg.event, np.ndarray):
-        types = [str(event.type) for event in eeg.event]
-        latencies = [event.latency for event in eeg.event]
-    else:
-        # only one event - TypeError: 'mat_struct' object is not iterable
-        types = [str(eeg.event.type)]
-        latencies = [eeg.event.latency]
+    annotations = _read_annotations_eeglab(eeg)
+    types = annotations.description
+    latencies = annotations.onset
+
```

Comment:
basically the rest of this function does some black magic to go from Annotations to events. this function should be shared across all IO routines.